include/deal.II/distributed/grid_tools.h

```diff
@@ -136,6 +137,22 @@ namespace parallel
                                   const std::function<void (const typename MeshType::active_cell_iterator &, const DataType &)> &unpack);
 
     /**
+     * Exchange with all processors of the MPI communicator @p mpi_communicator the vector of bounding
+     * boxes @p local_bboxes.
+     *
+     * This function is meant to exchange bounding boxes describing the locally owned
+     * cells in a distributed triangulation obtained with the function
+     * GridTools::compute_mesh_predicate_bounding_box .
+     *
+     * The output vector' has length's size is number of processes of the MPI communicator:
```

Comment:
please fix the grammar.

Reply:
Sorry...that's really painful to read even for me...